src/content/site/rules.md

```diff
@@ -42,96 +34,101 @@ If somebody is already getting help, do not jump in the middle of the conversati
 
-## 3. No illegal or prohibited topics / Security and hacking
+### Don’t post controversial content
 
-### Do not discuss anything prohibited by law or a service’s Terms of Service (ToS)
+If you wish to discuss topics like politics, religion, philosophy and alike, head over to our sister server, Controversy Central.
 
-This includes, but is not limited to: piracy, video game cheats/hacks, hacking, web scraping, running automated tools against a website or service against the ToS, and so on. Basically anything that the website or service in question does not allow, we won’t either.
+## 4. Don't obstruct moderation
 
-If you are unsure if it’s a suitable topic to discuss, please contact ModMail.
+### Use only English
+We’re an English server and our staff is primarily English speaking. We cannot moderate what we cannot understand.
 
-### Do not talk about hacking, “offensive security”, or the like
+### Do not impersonate staff
 
-This includes, but is not limited to: reverse engineering and decompiling
+Don’t pretend to be staff; this includes but is not limited to: copying their profile picture, putting a hammer emoji in your name, having the same name as a staff member and pretending to be a member of the staff team.
 
-Light talk about preventing and best practices for coding with security in mind are allowed. Everyone should be aware of and make a good effort to write secure code, but discussions around how to bypass a system will not be allowed.
+### Follow the spirit of the rules and assume good-faith
 
-## 4. Use only english
+This speaks for itself: don’t be difficult or generally a pain in the ass by trying to find loop holes in the rules or skirt around them.
+
+### Do not discuss infractions in public
+
+Use ModMail if you want to discuss an infraction or mute: don’t do it anywhere else on the server.
 
-We’re an English server and our staff is primarily English speaking. We cannot moderate what we cannot understand.
 
 ## 5. Stay on topic
 
 ### Don’t derail constructive conversation
 
 Technical and language channels need to be kept on topic, meaning these channels are not the place to have random conversations or to post things unrelated to that specific channel. If there’s an on-going conversation taking place, do not jump in with something completely unrelated.
 
-### Don’t post controversial content
+### No life advice
 
-In addition, content that is not on topic for the server or against the content policy shouldn’t be posted anywhere. If you wish to discuss topics like politics, religion, philosophy and alike, head over to our sister server, Controversy Central.
+This includes, but is not limited to: relationship advice, depression or suicidal talk. We are not qualified to offer professional advice; it just does not belong here. Note that career-related advice is fine.
 
-## 6. Do not advertise
+### No recruitment or ask for free work
```

Comment:
Should this be "No recruitment or asking for free work" instead of "ask"?

Reply:
That sounds better!